File: 4.model/doenca.php
<?php

class Doenca
{
    private $nome;
    private $sintomas; 
    private $tratamentoPaliativo;

    /*public function __construct($nome = '', $sintomas = null, $tratamentoPaliativo = '')
    {
        $this->nome = $nome;
        $this->sintomas = $sintomas ?? new ArrayObject();
        $this->tratamentoPaliativo = $tratamentoPaliativo;
    }*/

    // GETTERS
    public function getNome()
    {
        return $this->nome;
    }

    public function getSintomas()
    {
        return $this->sintomas;
    }

    public function getTratamentoPaliativo()
    {
        return $this->tratamentoPaliativo;
    }

    // SETTERS
    public function setNome($nome)
    {
        $this->nome = $nome;
    }

    public function setSintomas(ArrayObject $sintomas)
    {
        $this->sintomas = $sintomas;
    }

    public function setTratamentoPaliativo($tratamentoPaliativo)
    {
        $this->tratamentoPaliativo = $tratamentoPaliativo;
    }
    
     public function __toString()
    {
        $sintomasStr = implode(', ', (array)$this->sintomas);
        return "Doença: {$this->nome}\nSintomas: {$sintomasStr}\nTratamento Paliativo: {$this->tratamentoPaliativo}";
    }
}

?>

File: 9.Java/doenca.java
import java.util.ArrayList;

public class doenca {
    private String nome;
    private ArrayList<String> sintomas;
    private String tratamentoPaliativo;
    
    public doenca() {
    	
    	
    }

    public doenca(String nome, ArrayList<String> sintomas, String tratamentoPaliativo) {
		super();
		this.nome = nome;
		this.sintomas = sintomas;
		this.tratamentoPaliativo = tratamentoPaliativo;
	}

	public String getNome() {
        return nome;
    }

    public ArrayList<String> getSintomas() {
        return sintomas;
    }

    public String getTratamentoPaliativo() {
        return tratamentoPaliativo;
    }
}